story/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
+
```

Comment:
The changes in this pull request are minimal, consisting only of the addition of two blank lines at the top of the `story/admin.py` file. While these changes do not affect the functionality, please consider the following suggestions:

1. Ensure the added blank lines comply with the project's coding style guide. Typically, unnecessary blank lines should be avoided.
2. Update the commit message to describe the changes more clearly, even if they are minor. For example, "Added blank lines for formatting improvement."

Other than that, everything looks good!